src/Raven.Studio/typescript/widgets/virtualGrid/checkedColumn.ts

```diff
@@ -0,0 +1,26 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Checked cell template used for toggling selection of a row in the virtual grid.
+ */
+class checkedColumn implements virtualColumn {
+    public readonly cellClass = "checked-cell";
```

Comment:
public is default in typescript so there is no need to write this. (we are following the default resharper code style suggestions except class names starts with lower case)